src/styles/_consultation.scss

```
@@ -0,0 +1,31 @@
+.consultation {
+  display: flex;
+  justify-content: space-between;
+
+  &__text {
+    width: 480px;
+  }
+
+  &__heading {
+    font-size: 32px;
+    line-height: 40px;
+
+    margin-bottom: 24px;
```

Для кращої читаємості коду, спочатку ставимо позиціонування і розміри блоку
```suggestion
    margin-bottom: 24px;
    
    font-size: 32px;
    line-height: 40px;
```